### PR TITLE
Extending the landing page based on previous sketches

### PR DESCRIPTION
**Pull request by Ainali**
Not adding button for data quality since I don't find a good target.

Cogwheel illustrations based on the CC 0 image https://pixabay.com/illustrations/media-internet-message-network-2082641

### CODE REVIEW

File: templates/landing.html [hunk 1]
```diff
@@ -44,6 +44,32 @@ <h2>Explore by topic</h2>
       </ul>
     </div>
   </section>
+  <section>
+    <div class="container">
+      <div class="half">
+        <img aria-hidden="true" src="community.svg" widht="261" height="161"/>
```
Comment (Abbe98) [inline templates/landing.html:50]: `widht`-> `width`

File: templates/landing.html [hunk 2]
```diff
@@ -44,6 +44,32 @@ <h2>Explore by topic</h2>
...
+    </div>
+  </section>
+    <section class="bg-grey">
+    <div class="container">
+      <div class="half">
+        <div>
+        <h2>Data quality through constant curation</h2>
+        <p>Based on Wikipedia's sister project Wikidata the data is constantly fact checked by a rigorous and dedicated community. With the help of tools that help monitor changes in data and entering data correctly, collectively the data can be kept up to date and accurate.</p>
+        </div>
+        <div class="center-text">
```
Comment (Abbe98) [inline templates/landing.html:66]: `widht`-> `width`

File: static/main.css [hunk 3]
```diff
@@ -285,6 +285,11 @@ table ul li {
     grid-template-columns: auto;
 }
 
+.half {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```
Comment (Abbe98) [inline static/main.css:290]: `grid-template-columns: 1fr 1fr;` should maybe be `grid-template-columns: 1fr;` and break to two columns in the 600px [media query](https://github.com/govdirectory/website/blob/f9030f08b925364bec676466f63515d09d17e9e2/static/main.css#L400-L421)?

File: static/community.svg [hunk 4]
```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (Abbe98) [inline static/community.svg:1]: For both the SVGs we could strip the Inkscape data(In Inkscape: Save As->Plain SVG).

**Review by Abbe98 — CHANGES_REQUESTED** [review]
Nice! Superb work on the images!

File: static/community.svg (hunk 4 above)
Comment (Ainali) [reply static/community.svg:1]: Fixed!

File: static/main.css (hunk 3 above)
Comment (Ainali) [reply static/main.css:290]: Hmm, I think the media query is beyond me. Do you mind fixing this?

**Review by Abbe98 — APPROVED** [review]
:star: Nice! I will squash merge!

**Comment by Abbe98** [comment]
Deploy in progress!

**Pull request merged**